package.json

```diff
@@ -10,12 +10,10 @@
   "scripts": {
     "lint": "eslint . --max-warnings 0",
     "check": "yarn lint && yarn test",
-    "test:rules": "tape ./rules/*.test.js",
-    "test:plugins": "tape ./plugins/**/*.test.js",
-    "test": "yarn test:rules && yarn test:plugins"
+    "test": "jest ./rules/*.test.js ./plugins/**/*.test.js"
   },
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=10.17.0"
```

Comment:
We probably want apps that still run Node 8 be able to use this package – does this change prevent them from doing so?

Reply:
I was wondering about this as well. What's a suitable version for tools like these?

Reply:
I'm getting this error when I try any version < 10

`error semver@7.1.3: The engine "node" is incompatible with this module. Expected version ">=10". Got "8.17.0"`

Not sure what semver is doing here but it looks like it needs node 10 😕

Reply:
Node 8 is EOL and no longer supported. sewing-kit - which is how most apps will consume this rather than directly [dropped support in December in v0.1.116.0](https://github.com/Shopify/sewing-kit/blob/master/CHANGELOG.md#01160---2019-12-16), and[[stylelint dropped support in january in v13](https://github.com/stylelint/stylelint/releases/tag/13.0.0).

Let's go for an earlier minimum version though, how about ">=v10.0.0" so we don't force consumers to upgrade too far. This is what sewing-kit itself uses: https://github.com/Shopify/sewing-kit/blob/master/packages/sewing-kit/package.json#L8

Reply:
Looks like the lowest we can go is 10.10.0 here. Are we ok with that?